Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -1958,17 +1958,17 @@ CoreMathMLEnabled:
 
 CrossDocumentViewTransitionsEnabled:
   type: bool
-  status: testable
+  status: stable
```

Comment:
Do we really think they are shippable? Perhaps Preview for a while?

Reply:
Nobody is using this on live sites really, since it's Chrome only and fairly new. I've only found two decent demo sites, both by chrome devrel.

I think enabling it so we can get as much coverage as possible is worthwhile, and hopefully result in new demos being created.